### DISCUSSION TITLE
Possible typo in version?

### DISCUSSION THREAD

**Discussion by HotCakeX** [microsoft/WindowsAppSDK#4897] [General] [answered]
In here

https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/stable-channel

i see this

<img width="686" alt="image" src="https://github.com/user-attachments/assets/e358164f-5a9b-495e-bd04-921660ef9cee">

<br>

but there is no mention of ver 1.6.3 on GitHub releases

<br>

![image](https://github.com/user-attachments/assets/30042cbc-2512-41e8-b432-81c5f7559a43)

**Comment by DarranRowe** [comment, marked as answer]
This is fine, and this is the format which has been used for a while.
The 1.6.3 is more of a user/developer readable moniker. But the actual package version uses the date it was created for the last segment. So, it was created on 2024 (24), November (11), 14th.
